### PR TITLE
No ticket - removing tailwind css

### PR DESCRIPTION
**Pull request by wkaspryk**
Removing tailwind css, adding custom buttons.
Some global styles/CSS resets needed to be reapplied using globals.scss and for each component CSS module.
Also cleaned up filter bar with the additional dropdown added.

### CODE REVIEW

File: src/dashboard/src/components/buttons/Button.tsx [hunk 1]
```diff
@@ -1,34 +1,33 @@
-import styles from './buttons.module.scss';
-
+import React from 'react';
 import Image from 'next/image';
+import styles from './buttons.module.scss';
 
 interface IButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
-  variant?: 'primary' | 'secondary' | 'success' | 'warn' | 'error' | 'info' | 'link';
+  variant?: 'primary' | 'secondary' | 'disabled';
```
Comment (Fosol) [inline src/dashboard/src/components/buttons/Button.tsx:6]: Make `disabled` a separate optional property instead`

File: src/dashboard/src/components/filter/Filter.tsx [hunk 2]
```diff
@@ -14,14 +14,15 @@ export const Filter: React.FC = () => {
   return (
     <div className={styles.filter}>
       <h1>Filter</h1>
-      <Select options={tenantOptions} label="Tenant" placeholder="Select tenant" />
+      <Select className={styles.dropdown} options={tenantOptions} label="Tenant" placeholder="Select tenant" />
```
Comment (Fosol) [inline src/dashboard/src/components/filter/Filter.tsx:17]: These should be styled by the component itself as much as possible.  We don't need to apply it here.  Remove `className={styles.dropdown}` as it'll already be part of the component.

File: src/dashboard/src/components/auth/AuthState.tsx [hunk 3]
```diff
@@ -35,22 +36,14 @@ export const AuthState: React.FC<IAuthState> = ({ showName }) => {
     return (
       <Row className="profile">
         {showName && <div>{session?.user.name}</div>}
-        <button onClick={() => keycloakSessionLogOut()} title="Sign out">
-          <div className="group flex flex-row gap-1 items-center text-white-100 hover:text-gray-500 active:text-gray-900 font-bold">
-            <FaSignOutAlt />
-            Sign out
-          </div>
-        </button>
+          <button className={styles.button} onClick={() => keycloakSessionLogOut()} title="Sign out"><FaSignOutAlt />&nbsp;Sign out</button>
```
Comment (Fosol) [inline src/dashboard/src/components/auth/AuthState.tsx:39]: This button should use our `Button` component.  Replace and remove custom styling here `className={styles.button}`

**Review by Fosol — CHANGES_REQUESTED** [review]

File: src/dashboard/src/components/buttons/Button.tsx (hunk 1 above)
Comment (wkaspryk) [reply src/dashboard/src/components/buttons/Button.tsx]: updated

File: src/dashboard/src/components/filter/Filter.tsx (hunk 2 above)
Comment (wkaspryk) [reply src/dashboard/src/components/filter/Filter.tsx]: removed className

**Review by Fosol — APPROVED** [review]

**Pull request merged**